src/main/java/resources/PersonResource.java

```diff
@@ -160,7 +160,7 @@ public Response updatePerson(Person newPerson) throws TdpRecruitmentPasswordStor
             logger.warn("Person with id => {} not found", newPerson.getId().toString());
             return Response.status(Response.Status.NOT_FOUND).build();
         }
-        if (!personDao.findByEmail(newPerson.getEmail()).isEmpty()) {
+        if (!personDao.findByEmail(newPerson.getEmail()).isEmpty() && !newPerson.getEmail().equals(user.get().getEmail())) {
```

Comment:
this endpoint is also used for updating user details. if he doesnt change the email, it would never allow him to update details